src/mudblazor/MudBlazor.Template/Components/Layout/MainLayout.razor

```diff
@@ -28,9 +29,79 @@
 
 @code {
     private bool _drawerOpen = true;
+    private bool _isDarkMode = true;
 
     private void DrawerToggle()
     {
         _drawerOpen = !_drawerOpen;
     }
-}
\ No newline at end of file
+
+    private void DarkModeToggle()
+    {
+        _isDarkMode = !_isDarkMode;
+    }
+
+    public MudTheme Theme => new MudTheme()
+    {
+        PaletteLight = LightPalette,
+        PaletteDark = DarkPalette,
+        LayoutProperties = new LayoutProperties()
+    };
+
+    private readonly PaletteLight LightPalette = new()
```

Comment:
let's use `_lightPalette`, as we use _camelCase for private properties above

Reply:
@ScarletKuro shouldn't readonly fields generally be TitleCase as opposed to regular fields which are _camelCase?

Reply:
No, `private` and `private readonly` always share the same style,and if you look at code MudBlazor codebase or the MS repo: https://github.com/search?q=repo%3Adotnet%2Faspnetcore%20private%20readonly&type=code that's always the case.
If it was `const` or `static`, then sure it can be PascalCase

Reply:
Thanks!  I've renamed to `_lightPalette` and `_darkPalette` and fixed `=>` to `=`